.claude/commands/open-pr.md

```
@@ -0,0 +1,52 @@
+
+2. **Stage deliberately.** Add specific files by name — never `git add -A` or `git add .`. Before committing, review `git status` for anything unexpected (secrets, stray build output, `.env` files) and double-check contents of anything suspicious.
+
+3. **Commit inside the devcontainer, not on the host.** The pre-commit hook (`trufflehog` + `gitleaks` in `.husky/pre-commit`) calls `sudo`, which is disabled on the host — the commit will fail there. Run it through the container instead:
+
+   ```bash
+   cd docker
+   docker compose exec app sh -c "cd /workspaces/distopia && git commit -m '<message>'"
+   ```
+
+   Write a concise commit message focused on *why*, following the repository's existing commit style (`git log` for examples). End it with:
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)